examples/graphql-mutations/graphql_mutations.metatags

```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, graphql mutation
+description: This example demonstrates defining GraphQL mutation operations.
```

Comment:
```suggestion
description: This example demonstrates defining GraphQL mutation operation.
```

Reply:
Updated as: `This example demonstrates defining a GraphQL mutation operation.`.